### DISCUSSION TITLE
Device with input to monitor magnetic contact

### DISCUSSION THREAD

**Discussion by TRuHa83** [Koenkk/zigbee2mqtt#14428] [Devices]
Hello partners,

I am looking for a zigbee device by which I can monitor my magnetic contacts embedded in windows.

These can be battery powered or externally powered, but most importantly, compatible with Zigbee2MQTT.

All the devices I find are on the surface, looking bad aesthetically and always with the option of manipulation or blows.

I have thought about the XBee development boards but I don't see zigbee2MQTT support or code examples to give me an idea of the way to go.

Thank you so much for everything.
Greetings to all.

**Comment by Danit2** [comment]
maybe something like this?

https://a.aliexpress.com/_mOhq2sk

**Reply by TRuHa83** [reply]
I think it's just perfect.

I had seen these two custom products:
https://www.zigbee2mqtt.io/devices/ZigUP.html
https://www.zigbee2mqtt.io/devices/ZWallRemote0.html

But without a doubt, your option saves me the work of doing it, plus it is at a very good price.

Thank you very much.

**Reply by Danit2** [reply]
You can also use this project. 
https://ptvo.info

I love this firmware. Its so easy to use. 

With one of this:
https://a.aliexpress.com/_msbVVgo
Or this:
https://a.aliexpress.com/_mL8qrJA

There you can read up to 16 contacts at the same time. 

I use this to read the status of my pool. 😁
